apps/DisplayCluster/startdisplaycluster

```diff
@@ -12,20 +12,59 @@ import shlex
 import distutils.spawn
 import argparse
 
+# rtneuron vglrun env detection
+def using_virtualGL():
+    is_vglconnect = \
+        ('VGL_DISPLAY' in os.environ or 'VGL_CLIENT' in os.environ) and \
+        'SSH_CLIENT' in os.environ
+    return is_vglconnect
+
+# http://nullege.com/codes/search/distutils.spawn.find_executable
+def find_executable( executable, path=None ):
+    """Tries to find 'executable' in the directories listed in 'path'.
+
+    A string listing directories separated by 'os.pathsep'; defaults to
+    os.environ['PATH'].  Returns the complete filename or None if not found.
+    """
+    if path is None:
+        path = os.environ['PATH']
+    paths = path.split(os.pathsep)
+    base, ext = os.path.splitext(executable)
+
+    if(sys.platform == 'win32' or os.name == 'os2') and (ext != '.exe'):
+        executable = executable + '.exe'
+
+    if not os.path.isfile(executable):
+        for p in paths:
+            f = os.path.join(p, executable)
+            if os.path.isfile(f):
+                # the file exists, we have a shot at spawn working
+                return f
+        return None
+    else:
+        return executable
+
 parser = argparse.ArgumentParser()
 parser.add_argument("--config", help="The configuration file to load")
 parser.add_argument("--mpiargs", help="Extra arguments for the mpiexec command")
 parser.add_argument("--session", help="The session to load")
 parser.add_argument("--printcmd", help="Print the command without executing it",
                     action="store_true")
-parser.add_argument("--vglrun", help="Run the main application using vglrun",
```

Comment:
This argument is still needed. We typically run the main application in a VNC server using vglrun while the rest of the windows don't need it.